src/test/java/teammates/test/cases/browsertests/AdminEmailPageUiTest.java

```diff
@@ -59,6 +59,7 @@ private void testCompose() throws Exception {
 
         ______TS("send email - no subject");
```

Comment:
We could create an issue for this. Maybe you could help us.
There is no test case for sending email using `groupMode`.